.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -36,9 +39,21 @@ Tipo de Envio: <!--submission-type-->pre-envio<!--end-submission-type-->
 	- [ ] ligamientos con software de base de datos (database software bindings)
 	- [ ] datos geoespaciales (geospatial data)
 	- [ ] análisis de texto (text analysis)
+	
+     **Packages de Estadística**
+
+	- [ ] Rutinas Bayesianas y Monte Carlo (Bayesian and Monte Carlo Routines)
+	- [ ] Reducción de dimensionalidad, Análisis de Conglomerados y Aprendizaje no supervisado (Dimensionality Reduction, Clustering, and Unsupervised Learning)
```

Comment:
See `^3`.

Reply:
`^5:` The letter case is inconsistent. Maybe stick to either "Sentence case" or "Title Case".